pkgs/applications/window-managers/hyprwm/hyprland/wlroots.nix

```diff
@@ -2,38 +2,9 @@
 , hyprland
 , wlroots
 , lib
-, libdisplay-info
-, libliftoff
-, hwdata
 , enableNvidiaPatches ? false
 }:
-let
-  libdisplay-info-new = libdisplay-info.overrideAttrs (old: {
-    version = "0.1.1+date=2023-03-02";
-    src = fetchFromGitLab {
-      domain = "gitlab.freedesktop.org";
-      owner = "emersion";
-      repo = old.pname;
-      rev = "147d6611a64a6ab04611b923e30efacaca6fc678";
-      sha256 = "sha256-/q79o13Zvu7x02SBGu0W5yQznQ+p7ltZ9L6cMW5t/o4=";
-    };
-  });
 
-  libliftoff-new = libliftoff.overrideAttrs (old: {
-    version = "0.5.0-dev";
-    src = fetchFromGitLab {
-      domain = "gitlab.freedesktop.org";
-      owner = "emersion";
-      repo = old.pname;
-      rev = "d98ae243280074b0ba44bff92215ae8d785658c0";
-      sha256 = "sha256-DjwlS8rXE7srs7A8+tHqXyUsFGtucYSeq6X0T/pVOc8=";
-    };
-
-    NIX_CFLAGS_COMPILE = toString [
-      "-Wno-error=sign-conversion"
-    ];
-  });
-in
 wlroots.overrideAttrs
   (old: {
     version = "0.17.0-dev";
```

Comment:
Isn't this fully released by now?

Reply:
Hyprland uses a specific commit before the release, and the API of wlroots changes frequently.

Although I believe Hyprland will release a version that compatible with wlroots-0.17.0 soon (maybe

Reply:
@wineee not planned, we'll likely change the version to `0.18.0-dev` on the next Hyprland release.